### PR TITLE
made users posts display in profile and fixed serach and suggestions …

### PR DESCRIPTION
**Pull request by thajedster**
…component

### CODE REVIEW

File: client/src/components/Suggestions/index.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
 const Suggestions = props => {
-  const titles = props.results.map(title => <li key={title.id}>{title.title}</li>);
+  const titles = props.results.map(title => (
+    <Link to={"/post/" + title.id}>
+      <li key={title.id}>{title.title}</li>
```
Comment (charmata) [inline client/src/components/Suggestions/index.js]: the `<li>` should wrap the Link instead of the link text

**Review by charmata — CHANGES_REQUESTED** [review]

File: client/src/components/Suggestions/index.js (hunk 1 above)
Comment (thajedster) [reply client/src/components/Suggestions/index.js]: edited it!

Comment (thajedster) [reply client/src/components/Suggestions/index.js]: i got the links to properly work on the bio page now as well.

File: client/src/components/UserPost/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Userpost = props => {
+  const titles = props.post.map(title => <Link to={"/post/" + title.id}>{title.title}</Link>);
```
Comment (charmata) [inline client/src/components/UserPost/index.js]: this Link should also be wrapped in `<li>` with key

**Review by charmata — CHANGES_REQUESTED** [review]

File: client/src/components/UserPost/index.js (hunk 2 above)
Comment (thajedster) [reply client/src/components/UserPost/index.js]: i just removed user post and made it so that bio component can use suggestions to display the users posts

**Review by charmata — APPROVED** [review]

**Pull request merged**